app/src/main/res/layout/activity_sale_details.xml

```diff
@@ -28,113 +28,163 @@
                     android:layout_height="wrap_content"
                     android:clickable="false"
                     android:isScrollContainer="false"
-                    android:text="Edit"
+                    android:text="@string/edit"
                     android:visibility="invisible" />
 
                 <Button
                     android:id="@+id/deleteButton"
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
                     android:clickable="false"
-                    android:text="Delete"
+                    android:text="@string/delete"
                     android:visibility="invisible" />
             </LinearLayout>
 
-            <androidx.viewpager2.widget.ViewPager2
-                android:id="@+id/viewPagerImageSlider"
+            <RelativeLayout
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:paddingBottom="16dp"
+                android:orientation="vertical">
+
+                <androidx.viewpager2.widget.ViewPager2
+                    android:id="@+id/viewPagerImageSlider"
+                    android:layout_width="match_parent"
+                    android:layout_height="@dimen/viewPager2_height"
+                    android:background="@color/cardview_dark_background"
+                    android:visibility="invisible"/>
```

Comment:
Wouldn't it make more sense to set this to `gone`. Else if there are no images, there's just a blank space (it won't be `@color/cardview_dark_background`)

Reply:
It's in a relative layout with the loading image inside.